numba/npyufunc/parallel.py

```diff
@@ -41,7 +45,7 @@ def get_thread_count():
 NUM_THREADS = get_thread_count()
 
 
-def build_gufunc_kernel(library, ctx, innerfunc, sig, inner_ndim):
+def build_gufunc_kernel(library, ctx, info, sig, inner_ndim):
```

Comment:
I think the docstring for this function may need updating to reflect the changed return.

Reply:
Updated docstring on return value.